SimG4CMS/Calo/src/HGCalNumberingScheme.cc

```diff
@@ -50,6 +51,18 @@ uint32_t HGCalNumberingScheme::getUnitID(int layer, int module, int cell, int iz
       hgcons_.waferFromPosition(xx, pos.y(), layer, waferU, waferV, cellU, cellV, waferType, wt);
     }
     if (waferType >= 0) {
+      if (mode_ == HGCalGeometryMode::Hexagon8File) {
+        int type = hgcons_.waferType(layer, waferU, waferV, true);
+        if (type != waferType)
```

Comment:
is this if statement only supposed to apply to the `edm::LogVerbatim` line below? If so, I would put the if statement inside the `ifdef`.

Reply:
Thanks for pointing out this. I am correcting for that